packages/components/affix/__tests__/affix.spec.tsx

```diff
@@ -2,19 +2,13 @@ import { nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
 import { defineGetter, makeScroll } from '@element-plus/test-utils'
 import Affix from '../src/affix.vue'
+import type { VNode } from 'vue'
 
 let clientHeightRestore: () => void
 
-const _mount = (template: string) =>
-  mount(
-    {
-      components: {
-        'el-affix': Affix,
-      },
-      template,
-    },
-    { attachTo: document.body }
-  )
+const _mount = (template: VNode) => {
+  return mount(() => template, { attachTo: document.body })
```

Comment:
Can we remove this `() => template` and just use `mount(template)` here? Didn't see any difference here.

Reply:
I got no comments expect this one for now. The rest of this change seems legit to me.

Reply:
I want too, but `mount ` first argument must be `() => VNode`

![image](https://user-images.githubusercontent.com/39730999/155276284-010dd1d3-e1d7-4470-b09e-6f97a4bb7834.png)